seedDB/README.md

```diff
@@ -5,6 +5,25 @@ These files are MySQL database dump files for seeding a new instance of the cBio
 The database schema and cBioPortal release follows different numbering cycles since cBioPortal 1.5.0 and database schema 2.1.0. This means that the version numbers won't be identical. cBioPortal 1.9.0 with database schema 2.4.0 removed PDB annotations from the database.
 
 ## Latest seed database
+#### Seed database schema 2.8.2
+
+This schema is required for cBioPortal release versions:
+- **2.8.2**
```

Comment:
I don't understand what this is trying to say to the end user. In the instructions to the user for setting up a new portal deployment, https://docs.cbioportal.org/2.1-deployment/import-the-seed-database#import-the-cbioportal-seed-database they will run their version of cgds.sql through the mysql client first. This will create a schema at the version number which is correct for the version of cbioportal which they are intending to deploy. They they will be importing the seed database (which only contains data) into the schema they just set up. If we look at the content of the seed database (which tables are present) and compare that to various schemas, I think it is possible that these seed databases could be imported under different schemas. I think what we need to say here is something like "this seed database will install correctly under db_schema_version 2.3.2 through 2.8.2". Older versions of cbioportal which use a seed database outside of this range must be deployed with a different seed database.

Also, the cbioportal release number is not the same as the db_schema_version. I think we may be mixing up the two here. Where this says "cBioPortal release version" I bet it means the cbioportal version (now 2.2.0). To be very clear I would include a table showing cbioportal release numbers and the corresponding cgds.sql schema version numbers. So if someone is going to be running a version of the portal more recent than 2.2.0, then they will need to migrate this database schema to something more recent.

Reply:
Sorry I don't really understand this part. Should I just add "This seed database will install correctly under database schema_version 2.3.2 through 2.8.2. Older versions of cbioportal which use a seed database outside of this range must be deployed with a different seed database." under line 16?

Reply:
updated